katas/content/complex_arithmetic/complex_multiplication/index.md

```diff
@@ -0,0 +1,27 @@
+**Inputs:**
+
+1. A complex number $x = a + bi$.
+2. A complex number $y = c + di$.
+
+**Goal:**
+Return the product of x and y. That is $x * y = z = g + hi$.
+
+Revise the following Q# code to return the product of x and y.
+
+<details>
+  <summary><b>Need a hint?</b></summary>
+  
+* Use the Complex data type defined in the Q# math library. For example, $x = a + bi$:
```

Comment:
This information should be introduced once above, does not need to be repeated in each hint